register_enclave_direct.sh

```
@@ -0,0 +1,54 @@
+    echo "Example: $0 0x872852f77545c86a8bd9bdb8adc9e686b8573fc2a0dab0af44864bc1aecdaea9 0x2b70e34684d696a0a2847c793ee1e5b88a23289a7c04dd46249b95a9823367d9 0x86775ced1fdceae31d090cf48a11b4d8e4a613a2d49f657610c0bc287c8f0589 weather WEATHER"
+    exit 1
+fi
+
+ENCLAVE_PACKAGE_ID=$1
+EXAMPLES_PACKAGE_ID=$2
+ENCLAVE_CONFIG_OBJECT_ID=$3
+MODULE_NAME=$4
+OTW_NAME=$5
+
+echo 'Building CLI tool for direct attestation...'
+cd src/get_attestation_cli
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)